db/seeds.rb

```diff
@@ -115,9 +115,9 @@
 		phone: "206-621-0945",
 		website: "www.stfrancishouseseattle.org"
 	})
-StFrancisHouse.institution_detail = InstitutionDetail.new({
-		hours: "No Hours Set"
-	})
+# StFrancisHouse.institution_detail = InstitutionDetail.new({
+# 		hours: "No Hours Set"
+# 	})
```

Comment:
If you're going to remove the InstitutionDetail object then clear all of the Institutions out of the file at well. Otherwise we're going to have code failures if anyone runs the seeds because the objects aren't initialized for each class.